lms/templates/courseware/courseware.html

```diff
@@ -14,6 +15,11 @@
 </%def>
 
 <%block name="bodyclass">view-in-course view-courseware courseware ${course.css_class or ''}</%block>
+
+<%block name="js_overrides">
```

Comment:
I'm going to be moving this portion to main.html as part of my work on PERF-265.